.github/workflows/build.yml

```diff
@@ -11,32 +11,81 @@ jobs:
         include:
           - name: Ubuntu 18.04 (gcc)
             os: ubuntu-18.04
-            cmake_args: -DFAAD=ON -DKEYFINDER=ON -DLOCALECOMPARE=ON -DMAD=ON -DMODPLUG=ON -DWAVPACK=ON
+            cmake_args: >-
+              -DFAAD=ON
+              -DKEYFINDER=ON
+              -DLOCALECOMPARE=ON
+              -DMAD=ON
+              -DMODPLUG=ON
+              -DWAVPACK=ON
+            cmake_generator: Unix Makefiles
             ctest_args:
-            ccache_path: ~/.ccache
+            compiler_cache: ccache
+            compiler_cache_path: ~/.ccache
             cpack_generator: DEB
+            buildenv_basepath: /home/runner/buildenv
             buildenv_script: tools/ubuntu_buildenv.sh
             artifacts_name: Ubuntu 18.04 DEB
             artifacts_path: cmake_build/*.deb
+            qt_qpa_platform: offscreen
```

Comment:
Isn't this common to all builds?

Reply:
I wish. Unfortunately it complains about not finding that plugin on windows, even though QT_QPA_PLATFORM_PLUGIN_PATH was set correctly. Possibly because we don't use DLLs, so we need to link it at build time?